### PR TITLE
py-spy: add page

### PR DESCRIPTION
**Pull request by dmbaturin**
py-spy is a sampling profiler for Python programs.

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 0.3.10

### CODE REVIEW

File: pages/common/py-spy.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+> A sampling profiler for Python programs.
+> More information: <https://github.com/benfred/py-spy>.
+
+- Show a live view of the functions that take the most execution time, in a running process:
+
+`py-spy top --pid {{pid}}`
+
+- Start a program and show a live view of the functions that take the most execution time:
+
+`py-spy top -- python {{script file}}`
```
Comment (blueskyson) [inline pages/common/py-spy.md:12]:
```suggestion
`py-spy top --python {{path/to/file.py}}`
```

File: pages/common/py-spy.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# py-spy
+
+> A sampling profiler for Python programs.
+> More information: <https://github.com/benfred/py-spy>.
+
+- Show a live view of the functions that take the most execution time, in a running process:
```
Comment (blueskyson) [inline pages/common/py-spy.md:6]:
```suggestion
- Show a live view of the functions that take the most execution time of a running process:
```

**Review by blueskyson — COMMENTED** [review]
Thank you for contribution. Here are some suggestions.

File: pages/common/py-spy.md (hunk 1 above)
Comment (dmbaturin) [reply pages/common/py-spy.md]: Done.

File: pages/common/py-spy.md (hunk 2 above)
Comment (dmbaturin) [reply pages/common/py-spy.md]: Also done.

**Review by blueskyson — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @dmbaturin

**Pull request merged**